src/CMakeLists.txt

```
@@ -591,6 +592,26 @@ if(ALICEVISION_BUILD_SFM)
   endif()
 endif()
 
+# ==============================================================================
+# AprilTag
+# ==============================================================================
+# - optional, only external and enabled only if ALICEVISION_USE_APRILTAG is ON
+# ==============================================================================
+set(ALICEVISION_HAVE_APRILTAG 0)
+
+if(ALICEVISION_BUILD_SFM)
+  if(NOT ALICEVISION_USE_APRILTAG STREQUAL "OFF")
+    find_package(apriltag CONFIG)
```

maybe add the version 3.0 to make sure we use the latest version of the algo
```suggestion
    find_package(apriltag 3.0 CONFIG)
```

---

I tried this, but sadly the apriltagConfig.cmake from the AprilTag installation does not contain version information, making this fail. I raised an issue at the AprilTag repo.

---

Link to corresponding issue: https://github.com/AprilRobotics/apriltag/issues/116